src/readme.html

```diff
@@ -3,96 +3,96 @@
+			<li>If for some reason this doesn&#8217;t work, don&#8217;t worry. It doesn&#8217;t work on all web hosts. Open up <code>wp-config-sample.php</code> with a text editor and fill in the database connection details.</li>
 			<li>Save the file as <code>wp-config.php</code> and upload it.</li>
-			<li>Open <span class="file"><a href="wp-admin/install.php">wp-admin/install.php</a></span> in your browser.</li>
+			<li>Open <span class="file"><a href="wp-admin/install.php">wp-admin/install.php</a></span> in the browser.</li>
 		</ol>
 	</li>
-	<li>Once the configuration file is set up, the installer will set up the tables needed for your blog. If there is an error, double check your <code>wp-config.php</code> file, and try again. If it fails again, please go to the <a href="https://forums.classicpress.net/c/support" title="ClassicPress support">support forums</a> with as much data as you can gather.</li>
+	<li>Once the configuration file is set up, the installer will set up the tables needed for the website. If there is an error, double check the <code>wp-config.php</code> file, and try again. If it fails again, please go to the <a href="https://forums.classicpress.net/c/support" title="ClassicPress support">support forums</a> with as much data as you can gather.</li>
 	<li><strong>If you did not enter a password, note the password given to you.</strong> If you did not provide a username, it will be <code>admin</code>.</li>
-	<li>The installer should then send you to the <a href="wp-login.php">login page</a>. Sign in with the username and password you chose during the installation. If a password was generated for you, you can then click on &#8220;Profile&#8221; to change the password.</li>
+	<li>The installer should then go to the <a href="wp-login.php">login page</a>. Sign in with the username and password you chose during the installation. If a password was generated, it is possible to change it by clicking on the username in the top right corner after signing in. It is also possible to reach this Profile page by navigating to &#8220;Users&#8221; in the sidebar navigation and then selecting &#8220;Your Profile&#8221; from the sub-menu.</li>
 </ol>
 
 <h2>Updating</h2>
 <h3>Using the Automatic Updater</h3>
-<p>If you are updating from version 2.7 or higher, you can use the automatic updater:</p>
+<p>If you are updating from beta version 1.0.0 or higher, you can use the automatic updater:</p>
 <ol>
 	<li>Open <span class="file"><a href="wp-admin/update-core.php">wp-admin/update-core.php</a></span> in your browser and follow the instructions.</li>
-	<li>You wanted more, perhaps? That&#8217;s it!</li>
+	<li>That&#8217;s it!</li>
 </ol>
 
-<h3>Updating Manually</h3>
+<h3>Updating Manually via (S)FTP or the admin panel of the web host (such as cPanel)</h3>
```

Comment:
This is very long for a heading.  I'd move the added text out into a `p` tag.

Reply:
You're right. moved to its own paragraph.